chatmessage/test_chatmessage.py

```diff
@@ -0,0 +1,49 @@
+from .models import Chatmessage
+from account.models.professional import Professional
+from account.models.client import Client
+import pytest
+
+
+@pytest.mark.django_db
+def test_chat_response_code(client):
+    data = {'contact_id': 1, 'sender_id': 3, 'sender_type': 'P'}
+    response = client.get('/chatmessage/', data=data)
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_send_message(client):
+    professional1 = Professional.objects.all()[0]
+    client1 = Client.objects.all()[0]
+
+    contact_id = client1.client_id
+    sender_id = professional1.professional_id
+    sender_type = 'P'
+
+    msg = {'msg_sent': 'Test message!'}
```

Comment:
@patrisiakaplun 
Why is it needed? Is it chat with the professional?

Reply:
I'm not really sure I understand what exactly the question is about, could you clarify?